src/CameraControls.ts

```diff
@@ -239,595 +242,604 @@ export class CameraControls extends EventDispatcher {
 			three: ACTION.TOUCH_TRUCK,
 		};
 
-		if ( this._domElement ) {
-
-			const dragStartPosition = new THREE.Vector2() as _THREE.Vector2;
-			const lastDragPosition = new THREE.Vector2() as _THREE.Vector2;
-			const dollyStart = new THREE.Vector2() as _THREE.Vector2;
-			const elementRect = new THREE.Vector4() as _THREE.Vector4;
-
-			const truckInternal = ( deltaX: number, deltaY: number, dragToOffset: boolean ): void => {
-
-				if ( isPerspectiveCamera( this._camera ) ) {
-
-					const camera = this._camera;
-
-					const offset = _v3A.copy( camera.position ).sub( this._target );
-					// half of the fov is center to top of screen
-					const fov = camera.getEffectiveFOV() * THREE.MathUtils.DEG2RAD;
-					const targetDistance = offset.length() * Math.tan( fov * 0.5 );
-					const truckX    = ( this.truckSpeed * deltaX * targetDistance / elementRect.w );
-					const pedestalY = ( this.truckSpeed * deltaY * targetDistance / elementRect.w );
-					if ( this.verticalDragToForward ) {
+		// if ( this._domElement ) {
+		this._dragStartPosition = new THREE.Vector2() as _THREE.Vector2;
+		this._lastDragPosition = new THREE.Vector2() as _THREE.Vector2;
+		this._dollyStart = new THREE.Vector2() as _THREE.Vector2;
+		this._elementRect = new THREE.Vector4() as _THREE.Vector4;
+		// }
+
```

Comment:
Then this part going to be...
```suggestion
		this._elementRect = new THREE.Vector4();

		if ( this._domElement ) {

			const dragStartPosition = new THREE.Vector2() as _THREE.Vector2;
			const lastDragPosition = new THREE.Vector2() as _THREE.Vector2;
			const dollyStart = new THREE.Vector2() as _THREE.Vector2;
```

and scoped `cancelDragging`, `onMouseDown`, and other handers.

Reply:
@yomotsu if i may i would really like those to be extendable too. I actually already have the need for it :P Especially `onMouseDown`. May i ask why you want them scoped?

Reply:
I understand. However would you mind separating this PR to make the diff smaller?
- xxxInternal for this PR
- DOM event handlers for another

Reply:
ok i can wont have time to do it now though.